internal/config/config.go

```diff
@@ -136,7 +136,16 @@ func parseParameters(parametersStr string) (Parameters, error) {
 	parameters.VaultTLSConfig.TLSServerName = params["vaultTLSServerName"]
 	parameters.VaultTLSConfig.ClientCert = params["vaultTLSClientCertPath"]
 	parameters.VaultTLSConfig.ClientKey = params["vaultTLSClientKeyPath"]
-	parameters.VaultKubernetesMountPath = params["vaultKubernetesMountPath"]
+	k8sMountPath, k8sSet := params["vaultKubernetesMountPath"]
+	authMountPath, authSet := params["vaultAuthMountPath"]
+	switch {
```

Comment:
I've never seen this pattern of using switch statements this way in golang before, really cool!